src/index.css

```diff
@@ -1,10 +1,49 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+.Calc-body {
+  width: 50%;
+  height: 70%;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
+.Calc-display {
+  background-color: grey;
+  text-align: end;
+  padding: 25px 10px 0 0;
+  color: antiquewhite;
+  border: solid 1px rgb(197, 196, 196);
+}
+
+.Calc-btns,
+.Grey-btns1,
+.Grey-btns2 {
+  display: grid;
+}
+
+.Calc-display,
+.Calc-btn {
+  height: 50px;
+}
+
+.Calc-btns {
+  grid-template-columns: 75% 25%;
+}
+
+.Grey-btns {
+  background-color: rgb(233, 225, 225);
+}
+
+.Grey-btns1 {
+  grid-template-columns: 33.3% 33.3% 33.3%;
+}
+
+.Grey-btns2 {
+  grid-template-columns: 66.6% 33.3%;
+}
+
+.Orange-btns {
+  background-color: orange;
```

Comment:
- [ ] Your UI should match the original design including background colors. Kindly compare the attached screenshot and change the background images of target buttons to match that of the original design. :pray: In case you face difficulty getting the color value, you can use chrome extensions like color-picker. :pray: 

![image](https://user-images.githubusercontent.com/64894927/137954343-4ed541f3-1ad5-4781-8ceb-f216244a9d0a.png)       
![image](https://user-images.githubusercontent.com/64894927/137954377-3ae70c80-0c90-4f66-b2ef-397adac99ad2.png)